service/ziti-tunnel/cli/cmdlineParseFunctions.go

```diff
@@ -0,0 +1,164 @@
+package cli
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/dto"
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/service"
+)
+
+func convertToIdentityCli(id *dto.Identity) dto.IdentityCli {
+	return dto.IdentityCli{

```

Comment:
What did you need this?  the 'normal' Identity struct has all these fields already (and some extra)?